src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -31,68 +31,73 @@
-           to-network-amounts-by-chain             (send-utils/network-amounts-by-chain
-                                                    {:route          chosen-route
-                                                     :token-decimals token-decimals
-                                                     :native-token?  native-token?
-                                                     :receiver?      true})
-           sender-possible-chain-ids               (map :chain-id sender-network-values)
-           sender-network-values                   (if routes-available?
-                                                     (send-utils/network-amounts
-                                                      (if (= tx-type :tx/bridge)
-                                                        from-network-amounts-by-chain
-                                                        (send-utils/add-zero-values-to-network-values
-                                                         from-network-amounts-by-chain
-                                                         sender-possible-chain-ids)))
-                                                     (send-utils/reset-loading-network-amounts-to-zero
-                                                      sender-network-values))
-           receiver-network-values                 (if routes-available?
-                                                     (send-utils/network-amounts
-                                                      to-network-amounts-by-chain)
-                                                     (send-utils/reset-loading-network-amounts-to-zero
-                                                      receiver-network-values))
-           network-links                           (when routes-available?
-                                                     (send-utils/network-links chosen-route
-                                                                               sender-network-values
-                                                                               receiver-network-values))]
+   (let [chosen-route                            (:best suggested-routes-data)
+         {:keys [token collectible token-display-name
+                 receiver-network-values
+                 sender-network-values tx-type]} (get-in db [:wallet :ui :send])
+         token-decimals                          (if collectible 0 (:decimals token))
+         native-token?                           (and token (= token-display-name "ETH"))
+         to-network-amounts-by-chain             (send-utils/network-amounts-by-chain
+                                                  {:route          chosen-route
+                                                   :token-decimals token-decimals
+                                                   :native-token?  native-token?
+                                                   :receiver?      true})]
```

Comment:
the only change in this event is moving these definitions above the `enough-assets?` condition, so that we get the network information if there's no assets also.